client/wfnews-war/src/main/angular/src/app/components/active-wildfire-map/active-wildfire-map.component.ts

```diff
@@ -91,22 +91,42 @@ export class ActiveWildfireMapComponent implements OnInit, AfterViewInit {
       if (val.length > 2) {
         this.filteredOptions = [];
         self.searchLayerGroup.clearLayers();
-
+        let address = null;
+        let trimmedAddress = null;
+        let valueLength = null;
+        let valueMatch = null;
         this.placeData.searchAddresses(val).then(function (results) {
+
           if (results) {
             results.forEach((result) => {
-              let address = self.getFullAddress(result);
+              address = self.getFullAddress(result);
               result.address = address.trim();
               self.highlight(result);
-            });
+              trimmedAddress = result.address;
+              valueLength = val.length;
+              if (trimmedAddress != null) valueMatch = trimmedAddress.substring(0, valueLength);
+
+              if (address != null && valueLength != null && valueMatch != null &&
+                (val.toUpperCase() === address.toUpperCase() || val.toUpperCase() === valueMatch.toUpperCase())) {
+                  var index = results.indexOf(result);
```

Comment:
replace the var with a let or const (I assume const in this case as that value isn't changing)